content/blog/meet-our-new-hpe-dev-team-representative-at-hpe-technology-and-solution-summit-2023.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Meet our new HPE DEV Team Representative  at HPE Technology and Solution
+  Summit 2023!
+date: 2023-03-30T13:29:10.157Z
+author: Frederic Passeron
+authorimage: /img/frederic-passeron-hpedev-192.jpg
+disable: false
+tags:
+  - HPE TSS
+  - HPE DEV
+---
+![](/img/tss2023.png "HPE TECHNOLOGY AND SOLUTIONS SUMMIT 2023")
+
+HPE TECHNOLOGY AND SOLUTIONS SUMMIT 2023 (HPE TSS 2023) goes physical again this year in sunny Barcelona. After several virtual editions, you will have the pleasure to engage with HPE Developer Community representative. All our sessions will be hold by our new team member : Mathieu Losmede. 
+
+H﻿PE TSS is the largest HPE technical event taking place in EMEA. Whether you are an HPE Presales or Partner, you will have the opportunity to attend technical breakout, Hands-on Labs, as well as strategic keynotes, and also finally get to meet your peers in person!
+
+T﻿he final decision to turn the event into a physical one came late this year. As a result, the HPE DEV team will not be able to bring the full physical Hack Shack back in HPE TSS 2023. However, we still come with a few sessions, some Hands-on labs, and a few surprises...
```

Comment:
As a result, the HPE Developer Community team will...

Reply:
A few comments from me. Consider rephrasing this paragraph to:
"The decision to hold this event at a physical location came late this year. As a result, the HPE Developer Community team will be unable to bring the complete, well-loved Hack Shack to the event. And we will only be able to spare one member of our team to attend and run our sessions. However, we will be offering a number of sessions, hands-on labs, and a few surprises. So make sure you stop by our booth to see what we have to offer!"